lib/datasets.go

```diff
@@ -170,17 +170,31 @@ func (r *DatasetRequests) List(p *ListParams, res *[]dsref.VersionInfo) error {
 		}
 	}
 
-	// Convert old style DatasetRef list to VersionInfo list.
-	// TODO(dlong): Remove this and convert lower-level functions to return []VersionInfo.
-	infos := make([]dsref.VersionInfo, len(refs))
+	// Convert old style DatasetRef list to DatasetLogItem list.
+	// TODO(dlong): Remove this and convert lower-level functions to return []DatasetLogItem.
+	dataLogs := make([]DatasetLogItem, len(refs))
 	for i, r := range refs {
-		infos[i] = reporef.ConvertToVersionInfo(&r)
+		dataLogs[i] = ConvertToDatasetLogItem(&r)
 	}
-	*res = infos
+	*res = dataLogs
 
 	return err
 }
 
+// ConvertToVersionInfo converts an old style DatasetRef to the newly preferred DatasetLogItem
```

Comment:
This is here because I couldn't drop it in the repo/ref/converter due to dependency cycles. Ideally we should put the DataLogItem lower and not have it in base.

Reply:
Ah, sorry for the confusion, `List` shouldn't be affected by this change. Keep on using `VersionInfo` here, not `DatasetLogItem`.